vng_api_common/validators.py

```diff
@@ -311,40 +304,33 @@ class UniekeIdentificatieValidator:
 
     message = _("Deze identificatie bestaat al binnen de organisatie")
     code = "identificatie-niet-uniek"
+    requires_context = True
 
     def __init__(self, organisatie_field: str, identificatie_field="identificatie"):
         self.organisatie_field = organisatie_field
         self.identificatie_field = identificatie_field
 
-    def set_context(self, serializer):
-        """
-        This hook is called by the serializer instance,
-        prior to the validation call being made.
-        """
-        # Determine the existing instance, if this is an update operation.
-        self.instance = getattr(serializer, "instance", None)
-        self.model = serializer.Meta.model
-
-    def __call__(self, attrs: dict):
+    def __call__(self, attrs: dict, serializer):
+        instance = getattr(serializer, "instance", None)
         identificatie = attrs.get(self.identificatie_field)
         if not identificatie:
-            if self.instance:
+            if instance:
                 # In case of a partial update
-                identificatie = self.instance.identificatie
+                identificatie = instance.identificatie
             else:
                 # identification is being generated, and the generation checks for
                 # uniqueness
                 return
 
         organisatie = attrs.get(self.organisatie_field)
-        pk = self.instance.pk if self.instance else None
+        pk = instance.pk if instance else None
 
         # if we're updating an instance, setting the current values will not
         # trigger an error because the instance-to-be-updated is excluded from
         # the queryset. If either bronorganisatie or identificatie changes,
         # and it already exists, it will raise a validation error
         combination_exists = (
-            self.model.objects
+            serializer.Meta.model
```

Comment:
This validator doesn't work anymore and fails with this error (for Besluit model):
`AttributeError: type object 'Besluit' has no attribute 'exclude'` 

I think the reason is this line
```suggestion
            serializer.Meta.model.objects